main.tf

```diff
@@ -0,0 +1,30 @@
+#
+# S3 resources
+#
+data "template_file" "read_only_bucket_policy" {
+  template = "${file("${path.module}/policies/s3-read-only-anonymous-user.json")}"
```

Comment:
I think that we should use the `aws_iam_policy_document` resource to define this policy vs. templating JSON.